<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QieTVCloud</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#E9E9E9;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="MainLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F5F5F5;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>165</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>165</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_17" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>38</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>38</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_18" native="true">
                  <property name="styleSheet">
                   <string notr="true">border:none;
background: #F5F5F5;</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_15">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_16">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="bottomMargin">
                      <number>2</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="icon">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>75</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">border:none</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="name">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>15</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">font-family: PingFangSC-Regular;
font-size: 12px;
color: #5A5A5A;
text-align: center;
border:none</string>
                       </property>
                       <property name="text">
                        <string>QieTV</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QProgressBar" name="progressBar">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>5</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QProgressBar{
      border:none;
      background:#FFFFFF;
      border-radius:5px;
}
QProgressBar::chunk {
      background-color:#2EC17C;
      border-radius:5px;
}</string>
                       </property>
                       <property name="value">
                        <number>50</number>
                       </property>
                       <property name="textVisible">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QWidget" name="widget_19" native="true">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>15</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">border:none</string>
                       </property>
                       <layout class="QHBoxLayout" name="horizontalLayout_14">
                        <property name="spacing">
                         <number>0</number>
                        </property>
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_15">
                          <property name="spacing">
                           <number>0</number>
                          </property>
                          <item>
                           <widget class="QLabel" name="capacity">
                            <property name="styleSheet">
                             <string notr="true">font-family: PingFangSC-Regular;
font-size: 8px;
color: #BFBFBF;
text-align: left;
border:none</string>
                            </property>
                            <property name="text">
                             <string>5/50G</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QPushButton" name="expand">
                            <property name="styleSheet">
                             <string notr="true">font-family: PingFangSC-Regular;
font-size: 8px;
color: #BFBFBF;
text-align: right;
border:none</string>
                            </property>
                            <property name="text">
                             <string>扩容 </string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="loginOut">
                       <property name="styleSheet">
                        <string notr="true">font-family: PingFangSC-Regular;
font-size: 8px;
color: #4A90E2;
text-align: center;
border:none</string>
                       </property>
                       <property name="text">
                        <string>切换账号</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_20" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>37</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>37</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#E9E9E9;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_22">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_17">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none;
font-family: PingFangSC-Regular;
font-size: 12px;
color: #919191;
text-align: left;</string>
                  </property>
                  <property name="text">
                   <string>我的文件</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_21" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none;</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_18">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_19">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QListWidget" name="myFiles">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QListWidget{
	border:none;
	font-family: PingFangSC-Regular;
	font-size: 12px;
	color: #5A5A5A;
	text-align: left; 
	outline:0px;
}
QListWidget::Item{
	padding-top:5px;
	padding-bottom:4px;
	border:none;
}
QListWidget::item:selected{
	background:#E9E9E9; 
	border:none;
	font-family: PingFangSC-Regular;
	font-size: 12px;
	color: #5A5A5A;
	text-align: left;
}
</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none;
font-family: PingFangSC-Regular;
font-size: 12px;
color: #919191;
text-align: left;</string>
                  </property>
                  <property name="text">
                   <string>传输列表</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_22" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none;</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_20">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_21">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QListWidget" name="myMit">
                       <property name="styleSheet">
                        <string notr="true">QListWidget{
	border:none;
	font-family: PingFangSC-Regular;
	font-size: 12px;
	color: #5A5A5A;
	text-align: left; 
	outline:0px;
}
QListWidget::Item{
	padding-top:5px;
	padding-bottom:4px;
	border:none;
}
QListWidget::item:selected{
	background:#E9E9E9; 
	border:none;
	font-family: PingFangSC-Regular;
	font-size: 12px;
	color: #5A5A5A;
	text-align: left;
}</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_23" native="true">
                  <property name="styleSheet">
                   <string notr="true">border:none;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
